apps/api/openapi/openapi.json

```diff
@@ -4281,6 +4302,150 @@
         }
       }
     },
+    "/api/leaderboard": {
+      "get": {
+        "tags": [
+          "Leaderboard"
+        ],
+        "summary": "Get global leaderboard",
+        "description": "Get global leaderboard data across all relevant competitions",
+        "parameters": [
+          {
+            "in": "query",
+            "name": "type",
```

Comment:
technically, you can get the global leaderboard w/ a query param: `/leaderboard?type=trading`. i figured we can keep this in case we need thematic leaderboards in the future. since we only have one type of comp, `/leaderboard` just defaults to `?type=trading`